docs/docsite/rst/_ext/eol_version.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from pathlib import Path
+from typing import Any, Dict
+
+import yaml
+from docutils import nodes
+from jinja2 import Template
+from sphinx.transforms import Transform
+
+
+class EOLVersionCheck(Transform):
+    # pylint: disable=too-few-public-methods
```

Comment:
pro tip: try to use `disable-next` or put the suppression onto the exact line. bare `disable` turns the pylint check off until the end of the document, unlike what flake8 would do.